src/routes/transactions/__tests__/controllers/get-transaction-by-id.transactions.controller.spec.ts

```diff
@@ -391,12 +391,12 @@ describe('Get by id - Transactions Controller (Unit)', () => {
       .with('baseGas', baseGas)
       .with('confirmations', confirmations)
       .build();
-    const replacementTxConfirmations = [confirmationBuilder().build()];
-    const replacementTx = multisigTransactionBuilder()
-      .with('confirmations', replacementTxConfirmations)
+    const rejectionTxConfirmations = [confirmationBuilder().build()];
```

Comment:
I've just renamed `replacement*` to `rejection*` in this file, I think it's more meaningful and makes the logic easier to follow.